code/procs/jobprocs.dm

```diff
@@ -556,7 +556,7 @@
 	else if (src.traitHolder && src.traitHolder.hasTrait("loyalist"))
 		trinket = new/obj/item/clothing/head/NTberet(src)
 	else if (src.traitHolder && src.traitHolder.hasTrait("petasusaphilic"))
-		var/picked = pick(childrentypesof(/obj/item/clothing/head) - (typesof(/obj/item/clothing/head/bighat)) - (typesof(/obj/item/clothing/head/helmet/space/syndicate))) //IM A MONSTER DONT LOOK AT ME. NOOOOOOOOOOO
+		var/picked = pick(!hat_blacklist)
```

Comment:
pick from the negation of a list of types?

Reply:
does `var/picked = pick(childrentypesof(/obj/item/clothing/head) - (hat_blacklist))` seem right? to use instead?

Reply:
Why is not concretetypesof and the clothing/head abstract